src/utils/rojo-lock-manager.ts

```
@@ -0,0 +1,133 @@
+
+	// Check if process is still running
+	const isRunning = await isProcessAlive(pid);
+	if (!isRunning) {
+		log.warn(`Cleaned up stale Rojo lockfile (PID ${pid} no longer running)`);
+		await cleanupLockfile(lockPath);
+		return null;
+	}
+
+	// Guard against PID reuse: if the recorded port is free, the lock is stale.
+	if (await isPortAvailable(port)) {
+		log.warn(`Cleaned up stale Rojo lockfile (port ${port} is no longer bound to Rojo)`);
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)